src/WebJobs.Script/Workers/Http/HttpScriptInvocationResultExtensions.cs

```diff
@@ -78,19 +79,25 @@ private static object GetBindingValue(DataType? bindingDataType, object outputBi
 
         internal static object GetHttpOutputBindingResponse(string bindingName, IDictionary<string, object> outputsFromWorker)
         {
-            HttpOutputBindingResponse httpOut = new HttpOutputBindingResponse();
+            dynamic httpOutput = new ExpandoObject();
+
             if (outputsFromWorker.TryGetValue(bindingName, out object outputBindingValue))
             {
                 try
                 {
-                    httpOut = JsonConvert.DeserializeObject<HttpOutputBindingResponse>(outputBindingValue.ToString());
+                    HttpOutputBindingResponse httpOut = JsonConvert.DeserializeObject<HttpOutputBindingResponse>(outputBindingValue.ToString());
```

Comment:
if `outputBindingValue` is a string and deserializing to `HttpOutputBindingResponse` fails - this should return `outputBindingValue` right? If we dont have a test of this please do add tests
- We should handle simple strings/Json and also JObjects that look like ty[pical `HttpResponseMessage`

Reply:
Nit: Rename `httpOut` to `httpOutputBindingResponse`

Reply:
`if outputBindingValue is a string and deserializing to HttpOutputBindingResponse fails - this should return outputBindingValue right?`

In the previous implementation, if serialization failed, it would result in a json string with all values as "null". Following that, I updated the code and assigned null defaults to the keys in dynamic object and also added a test for the same.